tox.ini

```diff
@@ -59,6 +59,15 @@ commands =
 deps =
     flake8
 
+[testenv:sdist]
+basepython = python3.6
```

Comment:
Python 3.6 here, but Travis is 3.5: https://github.com/Pylons/pyramid/pull/3111/files#diff-354f30a63fb0907d4ad57269548329e3R23